test/ReverseProxy.Tests/Middleware/LoadBalancerMiddlewareTests.cs

```diff
@@ -71,19 +71,23 @@ public async Task Invoke_Works()
             httpContext.SetEndpoint(aspNetCoreEndpoint);
             Provide<ILoadBalancer, LoadBalancer>();
 
-            httpContext.Features.Set<IAvailableDestinationsFeature>(
-                new AvailableDestinationsFeature() { Destinations = new List<DestinationInfo>() { destination1, destination2 }.AsReadOnly() });
+            httpContext.Features.Set<IReverseProxyFeature>(
+                new ReverseProxyFeature()
+                {
+                    AvailableDestinations = new List<DestinationInfo>() { destination1, destination2 }.AsReadOnly(),
```

Comment:
Arrays work too, right?
```suggestion
                    AvailableDestinations = new[] { destination1, destination2 },
```